src/avitab/apps/MapApp.cpp

```diff
@@ -463,9 +464,9 @@ void MapApp::startCalibrationStep1() {
     messageBox = std::make_unique<avitab::MessageBox>(
             getUIContainer(),
                 "The current file is not calibrated.\n"
-                "You need to select two points that are far away from each other.\n"
-                "I will show a red square first - center it above a known location and enter its coordinates, then click the tracking icon again.\n"
-                "The square will then turn blue.\n"
+                "Select two points that are diagonally far away from each other.\n"
+                "I will show a red square first - center it above a known location and enter its coordinates (decimal or D'M'S or D'M), then click the tracking icon again.\n"
+                "If valid, the square will turn blue.\n"

```

Comment:
You're correct. D<degrees_symbol>M'S" is more correct, and D'M'S was my ugly invention.
I find calibration much easier outside of VR, pre-flight. Easier in positioning the cursor accurately and then also in typing the coordinates. As a result I use the physical keyboard. Since the degree symbol can be a pain to type, I avoided it and thought ' as a universal separator would be ok. However, taking into account your comments . . .  I've often seen DMS in a format where all the numbers are squashed together with no separator. I've also sometimes seen it with a space as separator, and think this is less error prone than no separator. So how about using a space as separator instead of ' ?

Reply:
Good point, I didn't consider international keyboard layout. It's easy to type in my layout.

I think space is a good compromise - avoids confusion and is easy to type.